src/simplewallet/simplewallet.cpp

```diff
@@ -101,6 +101,7 @@ enum TransferType {
 
 namespace
 {
+  const auto allowed_priority_strings = {"default", "unimportant", "normal", "elevated", "priority"};
```

Comment:
Looks like the GUI uses Low/Medium/High for unimportant, normal, elevated, and shows the fee multipliers in the UI as well.